internal/clients/teamSOSA/modules/AgentParameters.go

```diff
@@ -108,7 +108,7 @@ func clamp(value float64) float64 {
 
 func NewAgentParameters() *AgentParameters {
 	return &AgentParameters{
-		PlatonicTendency: 0.1,
+		PlatonicTendency: rand.Float64(),
```

Comment:
Sure, but we'll need some way of fixing this for the experimentation. You may find yourself having to pass this in as an argument to the function, as the distribution will likely be set in the server/main.go

This said, we need to make a fresh PR. This one is massively oversaturated. Can you implement this on a fresh `experimentation` branch after?